lib/std/os/windows.zig

```diff
@@ -28,6 +28,9 @@ pub const gdi32 = @import("windows/gdi32.zig");
 
 pub usingnamespace @import("windows/bits.zig");
 
+//version detection
+usingnamespace std.zig.system.windows;
```

Comment:
Prefer to avoid `usingnamespace` unless it's to set up the public declarations (as above). Here we should give it a name and refer to the functions therein via field access syntax.